.changeset/eleven-parrots-hope.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': major
+---
+
+Rename NotificationBanner component into NotificationCard.
```

Comment:
I'd mention that we plan to deprecate it in the future (similar to what you wrote in the PR description).

```suggestion
Renamed NotificationBanner component into NotificationCard.
```

Reply:
Done in 55993ac